.github/workflows/npm-publish.yml

```diff
@@ -18,21 +19,20 @@ jobs:
   NPM:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 20
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 14.x
-          registry-url: 'https://registry.npmjs.org'
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+          registry-url: https://registry.npmjs.org
```

Comment:
I think this broke because you removed the `with:`.

https://github.com/Mottie/github-reserved-names/actions/runs/10169214463/workflow